e2e/minio.yaml

```diff
@@ -21,18 +43,30 @@ metadata:
     name: minio
 spec:
   containers:
-  - name: minio
-    image: minio/minio
-    args:
-    - server
-    - /data
-    ports:
     - name: minio
-      containerPort: 9000
-      protocol: TCP
-    volumeMounts:
-    - name: data
-      mountPath: /data
+      image: minio/minio
+      args:
+        - server
+        - /data
+      ports:
+        - name: minio
+          containerPort: 9000
+          protocol: TCP
+      volumeMounts:
+        - name: data
+          mountPath: /data
+        - name: secret-volume
+          mountPath: /root/.minio/certs
   volumes:
-  - name: data
-    emptyDir: {}
+    - name: data
+      emptyDir: {}
+    - name: secret-volume
+      secret:
+        secretName: minio-cert
+        items:
+          - key: ca.crt
```

Comment:
`key: tls.crt`?
I'm not sure why this configuration has worked for the client with `endpointURL: https://minio.default.svc:9000`.

Reply:
> `key: tls.crt`?

Its ca.crt and tls.crt are identical in content in this environment, but I will change it to tls.crt because it may be misleading.

> I'm not sure why this configuration has worked for the client with endpointURL: https://minio.default.svc:9000

In minio, it seems that tls is activated by simply placing the certificate in a specific directory.
https://min.io/docs/minio/linux/operations/network-encryption.html#enabling-tls
Also, the port number used seems to be 9000, regardless of whether tls is enabled or not, unless you specify it with `--address`.
https://min.io/docs/minio/linux/reference/minio-server/minio-server.html#minio.server.-address

Reply:
fixed at https://github.com/cybozu-go/moco/pull/580/commits/e9dfe5430ce1a74560f94bf8aa63ce3f8c607ccf

For this test purpose, `CAs/public.crt` seemed unnecessary, so I deleted it.